src/scss/components/_login-options.scss

```diff
@@ -16,3 +16,8 @@
     @include margin(auto);
   }
 }
+
+.#{prefix(login-options__cosign)} {
+  padding: 0.6em;
```

Comment:
magic numbers are not themeable - can you please use a design token for this and set a value in the design tokens repository?

the following will do:

```scss
padding-block: var(--of-login-options-cosign-padding-block);
padding-inline: var(--of-login-options-cosign-padding-inline);
```

Also note the use of `block` and `inline` rather than top/bottom/left/right - as those naturally deal correctly with LTR/RTL and horizontal/vertical text writing modes.